apis/openstack-nova/src/main/java/org/jclouds/openstack/nova/v2_0/compute/functions/ImageToOperatingSystem.java

```diff
@@ -80,9 +80,9 @@ public OperatingSystem apply(final Image from) {
             is64Bit = matcher.group(2).equals("x64");
          }
       } else {
-         if (imageName.contains("Red Hat EL")) {
+         if (imageName.contains("Red Hat")) {
```

Comment:
It would be good to know (and I'd like to :)), but it looks like a safe change anyway?